bots/s3_block_public_access.py

```diff
@@ -0,0 +1,122 @@
+"""
+## s3_block_public_access
+What it does: turn on S3 Bucket Block public access
+Usage: AUTO: s3_block_public_access BlockPublicAcls_NewPUTRequests=<true|false>  BlockPublicPolicy_NewPUTRequests=<true|false>
```

Comment:
Just pass AUTO: s3_block_public_access <true|false|->  <true|false|->  
When - set for ignore the specific attribute if its optional so usage will be
AUTO: s3_block_public_access false false

Reply:
changed